docs/consumer/Receiver.md

```diff
@@ -0,0 +1,6 @@
+# Stake Receiver
```

Comment:
Wait is this the same as the Consumer? Don't recognize this contract and there isn't much detail here.